curriculum/challenges/english/01-responsive-web-design/css-piano/part-005.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 612e783368153d2061d57c05
+title: Part 5
+challengeType: 0
+dashedName: part-5
+---
+
+# --description--
+
+You also need to specify the character encoding for the HTML. Add a `meta` tag above your `title` element. On that new tag, set the `charset` attribute to `UTF-8`.
```

Comment:
Another place, I would be less specific in the instructions:
```suggestion
Add a `meta` tag to specify an accepted `charset` for the page.
```

The hints should do the work